pkg/uploader/kopia/progress.go

```diff
@@ -177,3 +177,7 @@ func (p *Progress) EstimationParameters() upload.EstimationParameters {
 func (p *Progress) Enabled() bool {
 	return true
 }
+
+func (p *Progress) GetIncrementalSize() int64 {
+	return p.estimatedTotalBytes - p.cachedBytes
```

Comment:
Why can we not show `cachedBytes` here to the user as lyndon suggested?

Reply:
Yes, we can, see https://github.com/vmware-tanzu/velero/pull/9357#issuecomment-3434952559